### PR TITLE
DONE registration

### LINKED ISSUE #2: Registration UI

### PR DESCRIPTION
**Pull request by gdmeridabit**
closes #2 

# card-20590387
# card-20590395
https://github.com/gdmeridabit/blog/projects/1#card-20590387
https://github.com/gdmeridabit/blog/projects/1#card-20590395

### CODE REVIEW

File: config/database.php [hunk 1]
```diff
@@ -36,23 +36,20 @@
...
+            'write' => [
+                'host' => '127.0.0.1'
+            ],
+            'driver'    => 'mysql',
+            'database'  => 'blog',
+            'username'  => 'root',
+            'password'  => '',
+            'charset'   => 'utf8',
+            'collation' => 'utf8_unicode_ci',
+            'prefix'    => '',
```
Comment (andy1dx) [inline config/database.php:51]: @gdmeridabit  Why not use ENV file for the connection data

File: database/migrations/2019_04_23_084559_posts.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('user_id',false,true);
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->string('title');
+            $table->string('description');
```
Comment (andy1dx) [inline database/migrations/2019_04_23_084559_posts.php]: This must be text 
```suggestion
            $table->longText('description');
```

File: database/migrations/2019_04_23_084647_likes.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     */
+    public function up()
+    {
+        Schema::create('likes', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('user_id')->unsigned();
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->bigInteger('post_id')->unsigned();
+            $table->foreign('post_id')->references('id')->on('posts');
+            $table->integer('count');
```
Comment (andy1dx) [inline database/migrations/2019_04_23_084647_likes.php]: This $table->integer('count') dont need because later will get from sql count

**Review by andy1dx — CHANGES_REQUESTED** [review]
@gdmeridabit 
Please fix 

Please make card from project become issues 
and add

closes #[issues number in description]

File: database/migrations/2019_04_23_084647_likes.php (hunk 3 above)
Comment (gdmeridabit) [reply database/migrations/2019_04_23_084647_likes.php]: good point, ok I will remove it

**Review by andy1dx — APPROVED** [review]

**Pull request merged**